### PR TITLE
Fix the update function

### PR DESCRIPTION
**Pull request by reaperzn**
… when ONLYINSTALLFROMPREP is set to yes and the updates have already been applied

### CODE REVIEW

**Comment by reaperzn** [comment]
This fixes #31

**Comment by reaperzn** [comment]
bump

**Comment by ruthealee** [comment]
Ran into this issue the other day. This fixes the specific issue that I encountered: user updated packages between prep and install phases.

I've tested the PR and it resolves the issue.  For cases where updates fail for other reasons I've submitted PR #56

File: auter.yumdnfModule [hunk 1]
```diff
@@ -58,7 +58,7 @@ function apply_updates() {
     RC=0
     if [[ $(ls -1 ${DOWNLOADDIR}/${CONFIGSET}/*.rpm | wc -l) -gt 0 ]]; then
       RPMS="${DOWNLOADDIR}/${CONFIGSET}/*.rpm"
-      RC=100
+      RC=$("${PACKAGE_MANAGER}" check-update $(find "${DOWNLOADDIR}"/"${CONFIGSET}"/*.rpm -printf "%f\n" | sed 's/.rpm$//g') &>/dev/null;echo $?)
```
Comment (piersc) [inline auter.yumdnfModule]: This is pretty obtuse:
- i think we need a comment to explain what's happening
- lets put the handling of $? on another line to make it more readable
- you might be able to use this to make it a bit cleaner:
```
rpm -qp ${RPMS}
```

**Pull request merged**